src/oneshot/timer_fd.rs

```diff
@@ -79,6 +130,21 @@ fn set_timer_value(fd: &RawTimer, timeout: time::Duration) {
     fd.set(new_value);
 }
 
+#[cfg(target_os = "android")]
+fn set_timer_value(fd: &RawTimer, timeout: time::Duration) {
+    let it_value = libc::timespec {
+        tv_sec: timeout.as_secs() as libc::time_t,
+        tv_nsec: libc::suseconds_t::from(timeout.subsec_nanos()),
+    };
+
+    let new_value = ffi::itimerspec {
```

Comment:
Make `itimerspec` alias in ffi to point to `libc::itimerspec` for non-android builds so that we wouldn't need separate android cfg here